src/SFA.DAS.Roatp.CourseManagement.Web/Models/Standards/ConfirmDeleteStandardViewModel.cs

```diff
@@ -0,0 +1,26 @@
+﻿using SFA.DAS.Roatp.CourseManagement.Application.Standards.Queries.GetStandardInformation;
+
+namespace SFA.DAS.Roatp.CourseManagement.Web.Models.Standards
+{
+    public class ConfirmDeleteStandardViewModel
+    {
+        public StandardInformationViewModel StandardInformation { get; set; }
+        public string BackUrl { get; set; }
+        public string CancelUrl { get; set; }
+
+        public static implicit operator ConfirmDeleteStandardViewModel(GetStandardInformationQueryResult source)
```

Comment:
A conversion from query to standard information already exists, should reuse it here. See the operator in the view model